backend/users-service/app/src/index.ts

```diff
@@ -1,7 +1,71 @@
-import { Elysia } from "elysia";
+import { Elysia, t } from 'elysia'
+import { swagger } from '@elysiajs/swagger'
+import { cors } from '@elysiajs/cors'
+import { helmet } from 'elysia-helmet'
+import jwt from '@elysiajs/jwt'
+import { PrismaClient } from '@prisma/client'
+import { logger } from '@bogeychan/elysia-logger'
+import { Role } from './constants'
 
-const app = new Elysia().get("/", () => "Users service").listen(80);
+const db = new PrismaClient()
+
+const app = new Elysia({ prefix: '/users' })
+  .use(cors())
+  .use(helmet())
+  .use(
+    jwt({
+      name: 'jwt',
+      secret: process.env.JWT_SECRET as string
+    })
+  )
+  .use(logger())
+  .use(swagger())
+  .get(
+    '/:id',
+    async ({ params: { id } }) =>
+      await db.user.findUnique({
+        where: {
+          id
+        },
+        include: {
+          questions: true
+        }
+      })
+  )
+  .post(
+    '/',
+    async ({ body: { avatarUrl, email } }) =>
```

Comment:
can add `name` as well? i can retrieve that value from github

Reply:
Resolved by adding `name` in `User` schema in c3fdb0918c28871b2f0848308f902c9e8ef931ff